.devcontainer/devcontainer.json

```diff
@@ -6,6 +6,8 @@
         "redhat.ansible",
         // Common
         "shardulm94.trailing-spaces",
+        // Git
+        "eamodio.gitlens",
```

Comment:
Also add to /workspaces/epiphany/.vscode/extensions.json

Reply:
It's there 😉